### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by jhack32**
Fixes Issue #32 .

Changes proposed in this pull request: 
This is just the form. I wanna get the OK from everyone before adding the functionality. 

EDIT:
The latest commit will be using formspree. They keep the latest 100 emails though.
https://formspree.io/

### CODE REVIEW

**Comment by dijonkitchen** [comment]
It makes sense to me!

**Review by aattsai — APPROVED** [review]

File: contact.html [hunk 1]
```diff
@@ -60,9 +60,10 @@
       </div>
       <div class="field">
         <label>Text</label>
-        <textarea name="text"></textarea>
+        <textarea name="message"></textarea>
       </div>
-      <button class="ui button" type="submit">Submit</button>
+      <input type="text" name="_gotcha" style="display:none" />
```
Comment (dijonkitchen) [inline contact.html:69]: What's this do? 

The example on their website was different: https://formspree.io/

Comment (jhack32) [reply contact.html:69]: Avoid spambots

**Review by dijonkitchen — CHANGES_REQUESTED** [review]
Let's add some of the other FormSpree advanced options like:

Reply to email, so we have their email and can respond quickly. There's no guarantee currently that they'll provide a contact number/email. 

Use _next to redirect to our page instead of FormSpree.

Default a subject so we can filter our email by email we get from the website.

File: contact.html [hunk 2]
```diff
@@ -52,17 +52,22 @@
         </h1>
       <div class="field">
         <label>Name</label>
-        <input type="text" name="name" placeholder="Name">
+        <input type="text" name="name" placeholder="Name" required />
       </div>
       <div class="field">
         <label>E-mail</label>
-        <input type="text" name="email" placeholder="E-mail">
+        <input type="text" name="email" placeholder="E-mail" required />
```
Comment (dijonkitchen) [inline contact.html]: Use "email" as the type for form validation: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

File: contact.html [hunk 3]
```diff
@@ -52,17 +52,22 @@
...
+      </div>
+      <div class="field">
+        <label>Subject</label>
+        <input type="text" name="_subject" placeholder="Subject" required />
```
Comment (dijonkitchen) [inline contact.html:63]: Do we want custom subjects or should we have a few options for them in a drop-down?

File: contact.html [hunk 4]
```diff
@@ -52,17 +52,22 @@
         </h1>
       <div class="field">
         <label>Name</label>
-        <input type="text" name="name" placeholder="Name">
+        <input type="text" name="name" placeholder="Name" required />
```
Comment (dijonkitchen) [inline contact.html:55]: Do we need to require their name? We can keep the form simpler without it or make it optional.

**Review by dijonkitchen — CHANGES_REQUESTED** [review]

**Comment by dijonkitchen** [comment]
Bump

File: contact.html (hunk 4 above)
Comment (jhack32) [reply contact.html:55]: Yeah, I think we should have a required name. Let's us know who it's from

File: contact.html (hunk 3 above)
Comment (jhack32) [reply contact.html:63]: I think it's okay to have custom subjects

**Review by aattsai — APPROVED** [review]

**Review by dijonkitchen — APPROVED** [review]

**Pull request merged**